R/Design-methods.R

```diff
@@ -1072,8 +1076,10 @@ setMethod("simulate",
       ## the reasons for stopping
       stopReasons <- lapply(resultList, "[[", "stop")
 
-      ## for dual simulations as it would fail in summary otherwise (for dual simulations reporting is not implemented)
-      stop_report <- matrix(TRUE, nrow = nsim)
+      # individual stopping rule results as matrix, labels as column names
+      stopResults <- lapply(resultList, "[[", "report_results")
+      stop_matrix <- as.matrix(do.call(rbind, stopResults))
```

Comment:
```suggestion
      stop_results <- lapply(resultList, "[[", "report_results")
      stop_report <- as.matrix(do.call(rbind, stop_results))
```
could be better maybe? because using snake_case consistently and using the same name as in the slot below

Reply:
similar for the other classes.